conn_with_mockserver_test.go

```diff
@@ -781,3 +781,556 @@ func TestSetLocalIsolationLevel(t *testing.T) {
 		t.Fatalf("begin isolation level mismatch\n Got: %v\nWant: %v", g, w)
 	}
 }
+
+func TestGenericConnectionState_GoogleSQL(t *testing.T) {
```

Comment:
This test shows the overall features of transactional connection state for GoogleSQL.